src/content/tools/mcp-server.md

```diff
@@ -140,18 +145,65 @@ properly by typing `/mcp` in the chat window in Agent mode.
 For more information see the official Gemini Code Assist
 documentation for [using agent mode][].
 
-[instructions]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
+[gca-setup]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
 [Gemini Code Assist]: https://codeassist.google/
 [Agent mode]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer
 [configure the Gemini CLI]: #gemini-cli
 [using agent mode]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
 
+### GitHub Copilot in VS Code
+
+:::note
+Support for the Dart MCP server in VS Code required
+v3.116 or later of the [Dart Code extension][].
+:::
+
+By default, the Dart-Code extension uses the
+[VS Code MCP API][] to register the Dart MCP server, as well
+as a tool to provide the URI for the active Dart Tooling Daemon.
+This automatically enables it for any tool or extension that
+uses these APIs for MCP configuration, such as GitHub Copilot.
```

Comment:
I'd delete this old addition. It's confusing and I think the previous sentence covers things nicely.